tokio/src/runtime/builder.rs

```diff
@@ -940,6 +940,9 @@ impl Builder {
     /// # }
     /// ```
     pub fn build(&mut self) -> io::Result<Runtime> {
+        #[cfg(all(tokio_unstable, feature = "usdt"))]
+        usdt::register_probes().map_err(|error| io::Error::new(io::ErrorKind::Other, error))?;
```

Comment:
The `usdt` documentation suggests that this should be exposed by libraries, but should only be invoked by the final binary.

Reply:
In the crate docs

https://github.com/oxidecomputer/usdt/blob/9a58fa8a730daabda2633382cb96eaa4f9d9a531/usdt/src/lib.rs#L253-L260

> Alternatively, library developers may call this function during some initialization routines required by their library. There is no harm in calling this method multiple times, even in concurrent situations.